std/complex.d

```diff
@@ -464,16 +464,16 @@ if (isFloatingPoint!T)
     assert(cmc.im == c1.im - c2.im);
 
     auto ctc = c1 * c2;
-    assert(approxEqual(abs(ctc), abs(c1)*abs(c2), EPS));
-    assert(approxEqual(arg(ctc), arg(c1)+arg(c2), EPS));
+    assert(isClose(abs(ctc), abs(c1)*abs(c2), EPS));
+    assert(isClose(arg(ctc), arg(c1)+arg(c2), EPS));
 
     auto cdc = c1 / c2;
-    assert(approxEqual(abs(cdc), abs(c1)/abs(c2), EPS));
-    assert(approxEqual(arg(cdc), arg(c1)-arg(c2), EPS));
+    assert(isClose(abs(cdc), abs(c1)/abs(c2), EPS));
+    assert(isClose(arg(cdc), arg(c1)-arg(c2), EPS));
 
     auto cec = c1^^c2;
-    assert(approxEqual(cec.re, 0.11524131979943839881, EPS));
-    assert(approxEqual(cec.im, 0.21870790452746026696, EPS));
+    assert(isClose(cec.re, 0.1152413197994, 1e-12));
+    assert(isClose(cec.im, 0.2187079045274, 1e-12));
```

Comment:
This might look like a reduction in accuracy, but the opposite is true. The original tests are essentialy `approxEqual(cec.re, 0.11, 0.0)` and `approxEqual(cec.im, 0.21, 0.0)`, the remaining digits where not used for comparison.